app/assets/javascripts/header.js

```diff
@@ -70,14 +68,10 @@ function setHeight() {
 
 function expandButton(event) {
   hideSmallTopNav();
-
-  if (event.data.value % 2 === 0) {
-    showExpandMe();
```

Comment:
can you please delete this method (`showExpandMe`) since you removed the only reference to it? there are also tests for it that you can remove in `HeaderSpec.js`.